### ISSUE TITLE
FR: List packages which miss the LICENSE variable

### ISSUE THREAD

**Issue by jonasstein** [mm1ke/gentoo-scripts#9]
`qgrep -NL LICENSE`

I fear there could be some false positives, because there could be for example an eclass introducing a license technically.

/usr/portage/eclass$ grep LICENSE *
reveals for example the freebsd eclass.

An output folder could be for example
`/LICENSE_missing/sort-by-maintainer/...`

**Comment by mm1ke** [comment]
I got some hints in irc about using the md5-cache and saw that actually no ebuild, except virtuals, misses the LICENSE variable. Generally you're right. Plenty of packages get their LICENSE via a eclass. First i was wondering why i found so much. Turns out i forgot to check for the eclasses...

However, this leaves me with the eclasses. Do they need a LICENSE aswell?
A nice list to start with would be: ```grep -L ^LICENSE *.eclass``` (in the eclass directory)
